AztecTests/Formatters/BlockquoteFormatterTests.swift

```diff
@@ -5,64 +5,69 @@ import Gridicons
 class BlockquoteFormatterTests: XCTestCase {
     func testApplyingBlockquoteOnFirstParagraph() {
         let textView = testTextView
-        let paragraphs = paragraphRanges(inString: textView.storage)
+        let storage = textView.storage
+        let paragraphs = paragraphRanges(inString: storage)
 
         let formatter = BlockquoteFormatter()
-        formatter.toggleAttribute(inTextView: textView, atRange: NSRange(location: 1, length: 1))
+        _ = formatter.toggle(in: storage, at: NSRange(location: 1, length: 1))
```

Comment:
In my modest opinion the `toggle` method definition should have `@discardableResult` added to it, so this assignment is not needed.

It sounds reasonable for some usages to let you ignore the returned range.